src/client/types.ts

```diff
@@ -24,3 +30,220 @@ type ClientRPCResponseResult<
 } & Omit<T, 'metadata'>;
 
 export type { ClientRPCRequestParams, ClientRPCResponseResult };
+// Agent messages
+export type StatusResultMessage = {
+  pid: number;
+} & NodeIdMessage &
+  PublicKeyMessage & {
+  clientHost: string;
+  clientPort: number;
+  agentHost: string;
+  agentPort: number;
+  certChainPEM: CertificatePEMChain;
+};
+// Identity messages
+export type IdentityMessage = {
+  providerId: string;
+  identityId: string;
+};
+export type ProviderSearchMessage = {
+  authIdentityId?: string;
+  identityId: string;
+  disconnected: boolean;
+  limit?: number;
+  searchTermList?: Array<string>;
+  providerIdList: Array<string>;
+};
```

Comment:
In stead of exporting where the type is defined, they need to be exported at the bottom. Refer to how the `nodes/agent/types.ts` did this.

Reply:
Resolved

Reply:
Is it? I don't see any changes for it.

Reply:
Not pushed yet